diff --git a/package/base-files/files/lib/functions/uci-defaults.sh b/package/base-files/files/lib/functions/uci-defaults.sh
index 5a8809d..9970eec 100644
--- a/package/base-files/files/lib/functions/uci-defaults.sh
+++ b/package/base-files/files/lib/functions/uci-defaults.sh
@@ -212,10 +212,10 @@ set network.lan='interface'
 set network.lan.ifname='$ifname'
 set network.lan.force_link=1
 set network.lan.type='bridge'
-set network.lan.proto='static'
-set network.lan.ipaddr='192.168.1.1'
-set network.lan.netmask='255.255.255.0'
-set network.lan.ip6assign='60'
+set network.lan.proto='dhcp'
+set network.lan6='interface'
+set network.lan6.ifname='@lan'
+set network.lan6.proto='dhcpv6'
 EOF
 }
 
diff --git a/target/linux/ar71xx/base-files/etc/uci-defaults/02_network b/target/linux/ar71xx/base-files/etc/uci-defaults/02_network
index f5c6865..f4c019c 100644
--- a/target/linux/ar71xx/base-files/etc/uci-defaults/02_network
+++ b/target/linux/ar71xx/base-files/etc/uci-defaults/02_network
@@ -235,10 +235,9 @@ uap-pro)
 	;;
 
 nanostation-m-xw)
-	ucidef_set_interfaces_lan_wan "eth0.1" "eth0.2"
+	ucidef_set_interfaces_lan "eth0.23"
 	ucidef_add_switch "switch0" "1" "1"
-	ucidef_add_switch_vlan "switch0" "1" "0t 5"
-	ucidef_add_switch_vlan "switch0" "2" "0t 1"
+	ucidef_add_switch_vlan "switch0" "23" "0t 1t 5t"
 	;;
 
 wrt160nl)
@@ -340,7 +339,7 @@ tube2h |\
 wndap360 |\
 mynet-rext |\
 wp543)
-       ucidef_set_interface_lan "eth0"
+       ucidef_set_interface_lan "eth0.23"
        ;;
 
 dir-505-a1)
@@ -470,6 +469,10 @@ wndr3700)
 	uci set network.@switch_port[-1].led='2'
 	;;
 
+nanostation-m)
+	ucidef_set_interface_lan "eth0.23 eth1.23"
+	;;
+
 *)
 	ucidef_set_interfaces_lan_wan "eth0" "eth1"
 	;;
