Index: openwrt/package/base-files/files/lib/functions/uci-defaults.sh
===================================================================
--- openwrt.orig/package/base-files/files/lib/functions/uci-defaults.sh	2015-05-07 20:24:52.718443032 +0200
+++ openwrt/package/base-files/files/lib/functions/uci-defaults.sh	2015-05-07 20:24:52.714442951 +0200
@@ -212,10 +212,10 @@
 set network.lan.ifname='$ifname'
 set network.lan.force_link=1
 set network.lan.type='bridge'
-set network.lan.proto='static'
-set network.lan.ipaddr='192.168.1.1'
-set network.lan.netmask='255.255.255.0'
-set network.lan.ip6assign='60'
+set network.lan.proto='dhcp'
+set network.lan6='interface'
+set network.lan6.ifname='@lan'
+set network.lan6.proto='dhcpv6'
 EOF
 }
 
Index: openwrt/target/linux/ar71xx/base-files/etc/uci-defaults/02_network
===================================================================
--- openwrt.orig/target/linux/ar71xx/base-files/etc/uci-defaults/02_network	2015-05-07 20:24:52.718443032 +0200
+++ openwrt/target/linux/ar71xx/base-files/etc/uci-defaults/02_network	2015-05-07 20:27:18.957381394 +0200
@@ -235,10 +235,9 @@
 	;;
 
 nanostation-m-xw)
-	ucidef_set_interfaces_lan_wan "eth0.1" "eth0.2"
+	ucidef_set_interfaces_lan "eth0.23"
 	ucidef_add_switch "switch0" "1" "1"
-	ucidef_add_switch_vlan "switch0" "1" "0t 5"
-	ucidef_add_switch_vlan "switch0" "2" "0t 1"
+	ucidef_add_switch_vlan "switch0" "23" "0t 1t 5t"
 	;;
 
 wrt160nl)
@@ -340,7 +339,7 @@
 wndap360 |\
 mynet-rext |\
 wp543)
-	ucidef_set_interface_lan "eth0"
+	ucidef_set_interface_lan "eth0.23"
 	;;
 
 dir-505-a1)
@@ -470,6 +469,10 @@
 	uci set network.@switch_port[-1].led='2'
 	;;
 
+nanostation-m)
+	ucidef_set_interface_lan "eth0.23 eth1.23"
+	;;
+
 *)
 	ucidef_set_interfaces_lan_wan "eth0" "eth1"
 	;;
